src/Microsoft.Extensions.DependencyModel/CompilationLibrary.cs

```diff
@@ -51,31 +51,31 @@ public class CompilationLibrary : Library
 
         public IEnumerable<string> ResolveReferencePaths()
         {
-            return ResolveReferencePaths(DefaultResolver);
+            var assemblies = new List<string>();
+
+            return ResolveReferencePaths(DefaultResolver, assemblies);
         }
 
-        public IEnumerable<string> ResolveReferencePaths(params ICompilationAssemblyResolver[] additionalResolvers)
+        public IEnumerable<string> ResolveReferencePaths(params ICompilationAssemblyResolver[] customResolvers)
```

Comment:
What do you think about exposing API to create `ICompilationAssemblyResolver` that extends DefaultResolver with custom ones (`CompilationLibrary.CreateCustomReolver`?) and then have `ResolveReferencePaths` overload that takes a single `ICompilationAssemblyResolver`?